### PR TITLE
Update free-courses-id.md

### PR DESCRIPTION
**Pull request by ahhzaky**
Add framework Codeigniter3

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) 

### Description 
Add framework Codeigniter3

### Why is this valuable (or not)?
you can learn from youtube.

### How do we know it's really free?
yes this is free from youtube

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order

### CODE REVIEW

File: free-courses-id.md [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
 * [PHP](#php)
+  * [Codeigniter](#codeigniter)
   * [Laravel](#laravel)
 * [Python](#python)
   * [Django](#django)
   * [Flask](#flask)
   * [PyTorch](#pytorch)
 * [Redis](#redis)
 
-
```
Comment (SethiPandi) [inline free-courses-id.md:27]: Add back this line break.

File: free-courses-id.md [hunk 2]
```diff
@@ -69,6 +69,7 @@
 * [Belajar Golang Basic Bahasa Indonesia](https://www.youtube.com/playlist?list=PLCZlgfAG0GXDztO-BFc9R5afhP26Dhsgm) - Sekolah Koding
 
 
+
```
Comment (SethiPandi) [inline free-courses-id.md:72]: Remove this line break.

File: free-courses-id.md [hunk 3]
```diff
@@ -145,13 +146,22 @@
 * [Tutorial CodeIgniter untuk pemula](https://www.youtube.com/playlist?list=PLCZlgfAG0GXCYh65VSFR2yzC7CuPBcAjt) - Sekolah Koding
 
 
+### Codeigniter
+
+* [Tutorial Codeigniter](https://www.youtube.com/watch?v=r2ONZZrYk8c&list=PLce3Eyp7oY9_5lzhkUtrV6ygriYcttMlg) - Framework Indonesia
+* [Tutorial Sistem Informasi Akademik dengan Codeigniter](https://www.youtube.com/watch?v=VaA_dmqs5so&list=PLce3Eyp7oY9_hXzGACf988F1ojvQlYmB0) - Framework Indonesia
+
+
+
```
Comment (SethiPandi) [inline free-courses-id.md:155]: Please remove just one line break from here. There should only be two lines.

File: free-courses-id.md [hunk 4]
```diff
@@ -145,13 +146,22 @@
...
 ### Laravel
 
 * [Belajar Laravel 7 Dari Awal](https://www.youtube.com/playlist?list=PLRKMmwY3-5MxfIKTn_wZ49XlplwHtz1AV) - Parsinta
 * [Membangun Web Profil Band dan Lirik dengan Laravel 8](https://www.youtube.com/playlist?list=PLRKMmwY3-5Mzoti-pT2MGuQERTd1_sm21) - Parsinta
 * [Tutorial Laravel 7.x Dasar](https://www.youtube.com/playlist?list=PLCZlgfAG0GXBucXejxeeqCe_NWZS-67q_) - Sekolah Koding
 
 
+
```
Comment (SethiPandi) [inline free-courses-id.md:163]: Please remove both line breaks you added here, there only should be two lines between sections.

**Review by SethiPandi — COMMENTED** [review]
You can check out the contribution guidelines (CONTRIBUTING.md) for more information on formatting and other guidelines for this repository. You can click on the failed check below and head to Travis CI to see exactly what was wrong with your PR and how to fix it!

**Comment by ahhzaky** [comment]
Hi there,

I've fixed it.

Thanks.

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator.

**Comment by ahhzaky** [comment]
Hi there,
I have changed the link to a "playlist" link.
Thanks.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**